### ISSUE TITLE
[BUG] Unable to load nvme module after using os.module Harvester config for installation

### ISSUE THREAD

**Issue by TachunLin** [harvester/harvester#3366]
**Describe the bug**

Adding the kvm and nvme setting value uder os.modules of Harvester configuration file https://docs.harvesterhci.io/v1.1/install/harvester-configuration/#osmodules
  ```
  os:
    modules:
      - kvm
      - nvme
  ```

After ipxe installation, check the `nvme` module did not correctly loaded 
  ```
  node1:~ # lsmod | grep kvm
  kvm_intel             270336  0
  kvm                   794624  1 kvm_intel
  irqbypass              20480  1 kvm
  node1:~ # lsmod | grep nvme
  ```

  ```
  node1:/oem # grep modules -A5 /oem/90_custom.yaml
      modules:
      - kvm
      - vhost_net
      - kvm
      - nvme
      systemctl:
  --
      modules: []
      systemctl:
        enable: []
        disable: []
        start: []
        mask: []
  --
      modules: []
      systemctl:
        enable: []
        disable: []
        start: []
        mask: []
  ```

**To Reproduce**
1. Prepare Harvester create and join configuration file and add the following os.modules settings
 ```
  os:
    modules:
      - kvm
      - nvme
  ```

2. Use config-create configuration to perform ipxe installation of the Harvester management node
3. Use config-join configuration to join the second node to cluster 
4. ssh to management node after installation complete
5. Run `lsmod | grep nvme` 
6. Run `grep modules -A5 /oem/90_custom.yaml` 

**Expected behavior**

The nvme os module should be loaded after using os.module configuration file to provision Harvester cluster

**Support bundle**

[supportbundle_63ba45a2-2ae7-44d3-ac1b-8e9c13065f1a_2023-01-13T09-53-24Z.zip](https://github.com/harvester/harvester/files/10410347/supportbundle_63ba45a2-2ae7-44d3-ac1b-8e9c13065f1a_2023-01-13T09-53-24Z.zip)

**Environment**
 - Harvester ISO version: `master-f3475489-head` (01/10)
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630): 2 nodes Harvester on bare metal machines

**Additional context**

**Comment by bk201** [comment]
I can repro this in the v1.1.1 deployment as well, not related to elemental bump.

**Comment by TachunLin** [comment]
Verified **fixed** on `master-1222750f-head` (04/25). Close this issue.

### Result
`nvme` module have been loaded after using kvm/nvme in os.module of config-create and config-join to provision Harvester cluster. 

```
node1:~ # lsmod | grep nvme
nvme                   53248  0
nvme_core             159744  1 nvme
nvme_common            24576  1 nvme_core
t10_pi                 16384  2 sd_mod,nvme_core

node1:~ # lsmod | grep kvm
kvm_intel             335872  0
kvm                  1052672  1 kvm_intel
irqbypass              16384  1 kvm
```

### Test Information
* Test Environment: 2 nodes harvester on bare machines
* Harvester version: `master-1222750f-head` (04/25)

### Verify Steps

1. Install Harvester with any nodes with os.modules kvm/nvme configured[^1]
2. Login to console, execute lsmod | grep vm
3. Output should contains: nvme, nvme_core, nvme_common, kvm, kvm_intel (if it's intel cpu)
